nautobot_device_lifecycle_mgmt/tables.py

```diff
@@ -239,18 +258,75 @@ class Meta(BaseTable.Meta):  # pylint: disable=too-few-public-methods
             "invalid",
             "no_software",
             "valid_percent",
+            "actions",
         ]
 
 
-class InventoryItemSoftwareValidationResultTable(BaseTable):
-    """Table for inventory item software validation report."""
+class DeviceSoftwareValidationResultListTable(BaseTable):
+    """Table for a list of device to software validation report."""
+
+    device = tables.Column(accessor="device", verbose_name="Device", linkify=True)
+    software = tables.Column(accessor="software", verbose_name="Current Software", linkify=True)
+    valid = tables.Column(accessor="is_validated", verbose_name="Valid")
+    last_run = tables.Column(accessor="last_run", verbose_name="Last Run")
+    run_type = tables.Column(accessor="run_type", verbose_name="Run Type")
+    valid_software = tables.TemplateColumn(
+        template_code="""{% for valid_software in record.valid_software.all %}
+                    <a href="/plugins/nautobot-device-lifecycle-mgmt/validated-software/{{ valid_software.id }}"
+                         %}">{{ valid_software }}
+                    </a>
+                    <br>
+                    {% endfor %}""",
+        verbose_name="Approved Software",
+        orderable=True,
+    )
 
-    part_id = tables.Column(accessor="inventory_item__part_id", verbose_name="Part ID")
-    total = tables.Column(accessor="total", verbose_name="Total")
-    valid = tables.Column(accessor="valid", verbose_name="Valid")
-    invalid = tables.Column(accessor="invalid", verbose_name="Invalid")
-    no_software = tables.Column(accessor="no_software", verbose_name="No Software")
+    class Meta(BaseTable.Meta):  # pylint: disable=too-few-public-methods
+        """Metaclass attributes of DeviceSoftwareValidationResultTable."""
+
+        model = DeviceSoftwareValidationResult
+        fields = ["device", "software", "valid", "last_run", "run_type", "valid_software"]
+        default_columns = [
```

Comment:
Do we not want to show `Approved Software` column by default?

Reply:
I can't remember why me and you didn't want it to show. I remember us talking about it and we only wanted it to show up in the cvs file.